packages/flutter/test/widgets/transform_test.dart

```diff
@@ -786,6 +786,101 @@ void main() {
 
     expect(tester.getBottomRight(find.byType(Container)), target.bottomRight(tester.getTopLeft(find.byType(Container))));
   });
+
+  testWidgets(
```

Comment:
Nit: I would usually turn this test into a `group` and then turn each `pumpWidget` below into a separate test.  That way you give each feature tested a chance to fail during one run and give a bit more info.  Up to you if you think it's useful or not here, though.

Reply:
Let's keep it as-is.